Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic.yml

```diff
@@ -1,6 +1,8 @@
-id: detonate_url_-_generic
+id: Detonate URL - Generic_patched
 version: -1
-name: Detonate URL - Generic
+contentitemexportablefields:
+  contentitemfields: {}
```

Comment:
Is the file formatted using SDK? I believe those fields should be automatically removed with the `format` command